apb.yml

```diff
@@ -1,5 +1,5 @@
 ---
-version: 1.0
+version: 1.2
```

Comment:
This should stay `1.0`. Tracking the version of APB in the label is a good approach.